tools/validator-client-scripts/README.md

```diff
@@ -0,0 +1,32 @@
+# NYM - CLI TOOL
+## Overview
+Set of simple CLI commands to perform basic actions, such as upload contract, bond mixnode, etc.
+Note: to work it requires rust 1.60+ 
+
+## The binary
+    - Build the binary 
+    - cargo build --release 
+    - this will compile the binary to "your/main/nym/path/target/release"
+    - the binary is named ./validator-client-scripts-rs
+
+## Executing commands
+    -./validator-scripts-rs --help
+    - will produce a list of it's capabilites 
+
+The binary requires mandatory OPTIONS:        
+```
+--config-env-file <CONFIG_ENV_FILE>
+--mixnet-contract <MIXNET_CONTRACT>      
+--mnemonic <MNEMONIC>                    
+--nymd-url <NYMD_URL>                    
+--vesting-contract <VESTING_CONTRACT>   
```

Comment:
I wonder, should contracts be mandatory if you don't need to interact with them? For example should you need to provide anything if you just want to send tokens between accounts (or just want to create a dummy account address)?

Reply:
These are opts as you've seen below :wink: